README.md

```diff
@@ -175,50 +235,66 @@ networks:
 Start it like this:
 
 ``` console
-$ docker-compose up
+$ docker compose up
```

Comment:
Are you sure? Here's what my console says:
```
$ docker compose up
docker: 'compose' is not a docker command.
See 'docker --help'
```
I think it's `docker-compose up`

Reply:
Nope, it's the new syntax. https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command